src/studio-header/Menu/Menu.jsx

```diff
@@ -0,0 +1,279 @@
+// This file was copied from edx/frontend-component-header-edx.
```

Comment:
There were a few modifications made to this file. The only changes were lint fixes and adding an `eslint-disable` line to line 163.